core/tests/protractor_desktop/navigation.js

```diff
@@ -20,91 +20,45 @@ var waitFor = require('../protractor_utils/waitFor.js');
 var ThanksPage = require('../protractor_utils/ThanksPage.js');
 var GetStartedPage = require('../protractor_utils/GetStartedPage.js');
 
-describe('Oppia static pages tour', function() {
-  var thanksPage = null;
+describe('Oppia landing pages tour', function() {
+  it('visits the Fractions landing page', async function() {
+    await browser.get('/fractions');
+    await waitFor.pageToFullyLoad();
 
-  beforeEach(function() {
-    browser.get(general.SERVER_URL_PREFIX);
-    waitFor.pageToFullyLoad();
-  });
-
-  it('visits the links in About dropdown', function() {
-    var LINKS_CLASS_NAMES = [
-      '.protractor-test-about-link',
-      '.protractor-test-get-started-link',
-      '.protractor-test-playbook-link'
-    ];
-
-    LINKS_CLASS_NAMES.forEach(function(className) {
-      var dropdown = element(by.css('.protractor-test-about-oppia-list-item'));
-      browser.actions().mouseMove(dropdown).perform();
-      dropdown.element(by.css(className)).click();
-      waitFor.pageToFullyLoad();
-    });
-  });
-
-  it('visits the donate link', function() {
-    element(by.css('.protractor-test-donate-link')).click();
-    waitFor.pageToFullyLoad();
-  });
-
-  it('visits the thanks for donating page', function() {
-    thanksPage = new ThanksPage.ThanksPage();
-    thanksPage.get();
-  });
-
-  it('visits the terms page', function() {
-    element(by.css('.protractor-test-terms-link')).click();
-    waitFor.pageToFullyLoad();
-  });
```

Comment:
Why aren't we testing navigating to these pages anymore?

Reply:
As we already have these tests (with a better testing approach around the end of this file.)

Reply:
In these tests, we don't have the "check whether the page is correct after loading" but the test written at the end of this file does have such a test, so I kept them and removed these tests.